test/unit/mesh-gateway-deployment.bats

```diff
@@ -460,7 +460,7 @@ key2: value2' \
       . | tee /dev/stderr \
       | yq '.spec.template.spec.containers[0]' | tee /dev/stderr )
 
-  [[ $(echo "${actual}" | yq -r '.command[2]' ) =~ '-service="mesh-gateway"' ]]
+  [[ $(echo "${actual}" | yq -r '.command[2]' ) =~ "-service=\"mesh-gateway\"" ]]
```

Comment:
Not that it matters, but I'm curious about this choice of quotes.

Reply:
it was breaking my editor syntax highlighting 😆